app/schemas/selectors/cinematic.js

```diff
@@ -230,6 +230,24 @@ const heroThangTypeOriginal = character => {
   return { enterOnStart, thang }
 }
 
+/**
+ * @param {ShotStup} shotSetup
+ * @returns {Object|undefined} camera properties with sensible defaults.
+ */
+const camera = shotSetup => {
+  if (!(shotSetup || {}).camera) {
+    return
+  }
+  const defaults = {
```

Comment:
We should make this a constant somewhere - maybe at the top of this file or in a cinematics model / store somewhere?

Reply:
Moved to top of CameraSystem. This is also where the initial camera position was moved to.

Great call outs. The camera was placed before there was a system, so refactoring the logic to where the camera system exists is a step in the right direction I believe.